symengine/lib/symengine_wrapper.pyx

```diff
@@ -2078,22 +2081,137 @@ class DenseMatrixIter(object):
 
     next = __next__
 
-Matrix = DenseMatrix
+cdef class MutableDenseMatrix(DenseMatrixBase):
+
+    def as_mutable(self):
+        return self.copy()
```

Comment:
`MutableDenseMatrix` is a Cython class. I'm not sure that it will work. Btw, you can just remove this method and keep the implementation from `DenseMatrixBase`